wgpu-types/src/lib.rs

```diff
@@ -1050,13 +1404,17 @@ impl<L> TextureDescriptor<L> {
     }
 }
 
+/// Kind of data the texture holds if it is a depth/stencil texture.
 #[repr(C)]
 #[derive(Copy, Clone, Debug, Hash, Eq, PartialEq)]
 #[cfg_attr(feature = "trace", derive(Serialize))]
 #[cfg_attr(feature = "replay", derive(Deserialize))]
 pub enum TextureAspect {
```

Comment:
This enum should have a name that reflects that it's only for depth/stencil textures, or should have a value for color aspects